web-common/src/features/metrics-views/metrics-internal-store.ts

```diff
@@ -47,7 +48,15 @@ timeseries: ""
 measures:
   - label: "Total Records"
     expression: "count(*)"
-dimensions: []
+dimensions:
```

Comment:
Whats the reason for adding a dummy dimension?

Reply:
Added this because the current semantic of having `[]` resulted in an error in the YAML editor. Also you don't know what are the keys you need to put in the `dimensions`. Now the first error the user sees is for wrong model name, followed by wrong column name which should give a path to the user to have the right spec.